Tests/BackendIntegrationTests/StoreKitIntegrationTests.swift

```diff
@@ -391,26 +382,36 @@ private extension StoreKit1IntegrationTests {
     }
 
     @discardableResult
-    func purchaseMonthlyOffering() async throws -> PurchaseResultData {
-        return try await Purchases.shared.purchase(package: self.monthlyPackage)
+    func purchaseMonthlyOffering(
+        file: FileString = #file,
+        line: UInt = #line
+    ) async throws -> PurchaseResultData {
+        let data = try await Purchases.shared.purchase(package: self.monthlyPackage)
+
+        try self.verifyEntitlementWentThrough(data.customerInfo,
```

Comment:
All purchases through this method now verify the entitlement for good measure (instead of most tests doing that repeatedly).